.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    browser: true,
```

Comment:
```suggestion
    node: true,
```
우테코의 테스트 실행 환경은 Node.js 14 이므로 `browser` 아닌 `node`가 되어야 할 것 같습니다!

Reply:
[25534e5](https://github.com/woowacourse-precourse/javascript-baseball/pull/462/commits/25534e5833e7c04b7cf0d909b69b62336565fa6c) 에 수정하였습니다. 피드백 감사합니다 ;)